### PR TITLE
List ensembles

### PR DESCRIPTION
**Pull request by mortalisk**
**Issue**
Resolves https://github.com/equinor/ert-storage/issues/148

### CODE REVIEW

File: ert_shared/dark_storage/endpoints/ensembles.py [hunk 1]
```diff
@@ -19,7 +19,17 @@ def post_ensemble(
...
     *, res: LibresFacade = Depends(get_res), ensemble_id: UUID
 ) -> js.EnsembleOut:
-    raise NotImplementedError
+    return js.EnsembleOut(
+        id=ensemble_id,
+        children=[],
+        parent=None,
+        experiment_id=get_id("experiment", "default"),
+        userdata={"name": get_name("ensemble", ensemble_id)},
+        size=42,
```
Comment (oysteoh) [inline ert_shared/dark_storage/endpoints/ensembles.py:28]: why 42? Is this a magic number? If it should stay - maybe extract as variable with a proper naming?

File: ert_shared/dark_storage/endpoints/experiments.py [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
 
 router = APIRouter(tags=["experiment"])
 
 
 @router.get("/experiments", response_model=List[js.ExperimentOut])
 def get_experiments(*, res: LibresFacade = Depends(get_res)) -> List[js.ExperimentOut]:
-    raise NotImplementedError
+    return [
+        js.ExperimentOut(
+            name="default",
```
Comment (oysteoh) [inline ert_shared/dark_storage/endpoints/experiments.py:17]: This name could be something else right? Should we have it hardcoded for the time being or should me make it work without hardcoding?

File: ert_shared/dark_storage/enkf.py [hunk 3]
```diff
@@ -1,11 +1,66 @@
...
+_ert: Optional[EnKFMain] = None
+_config: Optional[ResConfig] = None
+ids = {}
+
+
+def init_facade() -> None:
+    global _libres_facade
+    global _ert
+    global _config
+    # Assuming we are running poly_example and current working directory is poly_example
```
Comment (oysteoh) [inline ert_shared/dark_storage/enkf.py:23]: this comment is not valid anymore

File: tests/dark_storage/conftest.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+
+@pytest.fixture
+def dark_storage_client(dark_storage_app):
+    return TestClient(dark_storage_app)
+
+
+@pytest.fixture
+def env(monkeypatch):
+    monkeypatch.setenv("ERT_STORAGE_DATABASE_URL", "sqlite://")
```
Comment (oysteoh) [inline tests/dark_storage/conftest.py:30]: I don't see why we need the database_url set. We are running without _--enable-new-storage_ right? At least we should. Just running the test in ert-storage also creates a _storage_  folder in the runpath. I don't understand why this should be like that :thinking:

Comment (mortalisk) [reply tests/dark_storage/conftest.py:30]: I fixed the storage folder creation at least

**Comment by oysteoh** [comment]
Created an issue to follow up on the missing data for the ensemble here #2011

**Comment by oysteoh** [comment]
Created issue to follow up on the need of setting env varible ERT_STORAGE_DATABASE_URL in our tests in dark-storage here #2012

**Comment by oysteoh** [comment]
Created discussion issue #2013 regarding how to handle experiment in dark-storage

**Review by oysteoh — APPROVED** [review]
Consider this PR as good to go and a good basis for the rest of the features to be implemented regarding the dark-storage.

File: tests/dark_storage/conftest.py (hunk 4 above)
Comment (oysteoh) [reply tests/dark_storage/conftest.py:30]: yes, i can see the url is needed due to us importing ert-storage to verify compability between the two api's. Created an issue on this.

**Pull request merged**